themes/default/content/docs/pulumi-cloud/esc/faq.md

```diff
@@ -0,0 +1,43 @@
+---
+title_tag: "Pulumi ESC FAQs"
+meta_desc: Frequently asked questions about preview, GA, pricing and roadmap 
+title: FAQ
+h1: Pulumi ESC FAQs
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  pulumicloud:
+    parent: esc
+    weight: 6
+    identifier: faq
+---
+
+**Is it safe to use Pulumi ESC for my production stacks?**
```

Comment:
This questions makes me feel like the answer should be less about HA and more about security. Is it worth talking about the elements of security built into this service? Given that this is already discussed on the Pulumi Security Whitepaper, it may be worth linking to that at a minimum. I would still make a mention of something about security here like the following:

Yes, Pulumi ESC is served using [the same highly secure, highly available infrastructure](/security/pulumi-cloud-security-whitepaper.pdf) as the rest of Pulumi Cloud...

Reply:
updated! Thank you.

Reply:
updated!